com.unity.render-pipelines.high-definition/Runtime/Lighting/AtmosphericScattering/OpaqueAtmosphericScattering.shader

```diff
@@ -120,6 +120,7 @@ Shader "Hidden/HDRP/OpaqueAtmosphericScattering"
             Cull Off    ZWrite Off
             Blend One SrcAlpha // Premultiplied alpha
             ZTest Less  // Required for XR occlusion mesh optimization
+            ColorMask RGB 0 // Only write on the RGB channels for RT 0 (required to preserve alpha)
```

Comment:
instead of not writing in alpha channel, you should say what to do with alpha in the blend mode

Blend SrcFactor DstFactor, SrcFactorA DstFactorA: Same as above, but use different factors for blending the alpha channel.

https://docs.unity3d.com/Manual/SL-Blend.html

i.e in the blend mode of alpha just do nothing

Reply:
Done! 

The code path for PBR fog is always disabled for now (and this one is not using any blending), so I have tested it by manually enabling it it the code (and it was working in both R11G11B10 and 16-bit modes) .